pkg/mount/dag_steps.go

```diff
@@ -553,22 +551,6 @@ func (s *State) UKIMountBaseSystem(g *herd.Graph) error {
 	)
 }
 
-// UKIRemountRootRODagStep remount root read only.
-func (s *State) UKIRemountRootRODagStep(g *herd.Graph) error {
```

Comment:
this was not used anymore to remount as we cannot remount RO this soon if we are doing a fake sysroot change :D